project1-sps/script.js

```diff
@@ -1,4 +1,79 @@
+var NumPlayerWins = 0;
+var NumPlayerlosses = 0;
+var NumPlayerdraws = 0;
+
+var mode = "get user name";
+var UserName = "";
```

Comment:
Nice global variables here - but make sure that variable names are in lowerCamelCase (start with lowercase!)